src/core/github.js

```diff
@@ -5,10 +5,58 @@
  */
 
 import l10n from "core/l10n";
+import css from "deps/text!deps/balloon.css";
 import { pub } from "core/pubsubhub";
 
 export const name = "core/github";
 
+function enableFileIssuesOnSelect(issueBase){
+  const { body, head } = document;
+
+  const codeStyle = document.createElement("style");
+  codeStyle.textContent = css;
+  codeStyle.classList.add('removeOnSave');
+  head.appendChild(codeStyle);
+
+  var fileIssueLink = document.createElement('button');
+  fileIssueLink.setAttribute('data-balloon', 'file an issue related to the text selected');
+  fileIssueLink.setAttribute('data-balloon-pos', 'up');
+  fileIssueLink.setAttribute('id', 'respec-file-issue-btn');
+  fileIssueLink.classList.add('removeOnSave', 'respec-file-issue-hidden');
+  fileIssueLink.innerText = 'File Issue';
+  fileIssueLink.onclick = function () {
+    window.location.href = fileIssueLink.getAttribute('data-href');
+    fileIssueLink.classList.add('respec-file-issue-hidden');
+  };
+  body.appendChild(fileIssueLink);
+
+  body.addEventListener("mouseup", function () {
+    const selectionObj = window.getSelection();
+    const { top, right, left } = selectionObj.getRangeAt(0).getBoundingClientRect();
```

Comment:
So basically I now have `top, right and left`. So I can position the label as 
 - `y`: top
 - `x`: (right + left) / 2

But I can't find a way to change it to the css as it involves change in the `::before and ::after` pseudo css elements, fir which I have no idea of how to implement. @marcoscaceres, really need your help here :worried:

Reply:
I'm not sure why pseudo CSS elements should be manipulated, would you like to explain more for me?